.circleci/config.yml

```diff
@@ -0,0 +1,149 @@
+            .circleci/is-version-number-acceptable.sh
+
+  deploy_python2:
+    docker:
+      - image: python:2.7.14
+    environment:
+      PYPI_USERNAME: openfisca-bot  # Edit this value to replace it by your Pypi username
+      # PYPI_PASSWORD: this value is set in CircleCI's web interface; do not set it here, it is a secret!
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py2-{{ checksum "setup.py" }}
+
+      - run:
+          name: Check for functional changes
+          command: if .circleci/detect-functional-changes.sh ; then circleci step halt ; fi
+
+      - run:
+          name: Upload a Python package to Pypi
+          command: |
+            source /tmp/venv/country_template/bin/activate
+            .circleci/publish-python-package.sh
+
+      - run:
+          name: Publish a git tag
+          command: .circleci/publish-git-tag.sh
+
+  build_python3:
+    docker:
+      - image: python:3.6
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py3-{{ checksum "setup.py" }}
+
+      - run:
+          name: Create a virtualenv
+          command: |
+            mkdir -p /tmp/venv/country_template
+            python -m venv /tmp/venv/country_template
+            echo "source /tmp/venv/country_template/bin/activate" >> $BASH_ENV
+
+      - run:
+          name: Install dependencies
+          command: |
+            pip install --upgrade pip twine wheel
+            pip install --editable .[test] --upgrade
+          #  pip install --editable git+https://github.com/openfisca/country-template.git@BRANCH_NAME#egg=OpenFisca-Country-Template  # use a specific branch of OpenFisca-Country-Template
+
+      - save_cache:
+          key: v1-py3-{{ checksum "setup.py" }}
+          paths:
+            - /tmp/venv/country_template
+
+      - run:
+          name: Run tests
+          command: make test
+
+      - run:
+          name: Check version number has been properly updated
+          command: |
+            git fetch
+            .circleci/is-version-number-acceptable.sh
+
+  deploy_python3:
+    docker:
+      - image: python:3.6
+    environment:
+      PYPI_USERNAME: openfisca-bot  # Edit this value to replace it by your Pypi username
+      # PYPI_PASSWORD: this value is set in CircleCI's web interface; do not set it here, it is a secret!
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py3-{{ checksum "setup.py" }}
+
+      - run:
+          name: Check for functional changes
+          command: if .circleci/detect-functional-changes.sh ; then circleci step halt ; fi
+
+      - run:
+          name: Upload a Python package to Pypi
+          command: |
+            source /tmp/venv/country_template/bin/activate
+            .circleci/publish-python-package.sh
+
+workflows:
+  version: 2
+  build_and_deploy:
+    jobs:
+      - build_python2
+      - build_python3
+      - deploy_python2:
+          requires:
+            - build_python2
```

Comment:
I don't think we want to publish a Python2 package that does not pass Python3 builds. How would we manage a `23.1.2` tag that got published in Python2 but failed in Python3? We would then have to publish a `23.1.3` tag that would be available in both environments… So all version numbers could co-exist differently in each language version 😕

Reply:
OK, I misinterpreted a previous comment.